lib/perimeter_x.rb

```diff
@@ -77,56 +77,30 @@ def px_verify_request
             response.headers['Content-Type'] = 'text/html'
             render :html => html
           end
-        else # Mobile SDK
```

Comment:
this is incorrect.MobileSDK response reading is based on this json. The page property already contains the m=1 prop.

Reply:
fixed - returned to how it was before